package/MDAnalysis/lib/distances.py

```diff
@@ -101,6 +109,8 @@
    :class:`~MDAnalysis.core.groups.AtomGroup` or an :class:`np.ndarray`
 .. versionchanged:: 2.5.0
    Interface to the `distopia`_ package added.
+.. versionchanged:: 2.7.0
+    Distopia support greatly expanded.
```

Comment:
Indentation and more detail
```suggestion
   Distopia support greatly expanded (with distopia ≥ 0.3.0).
```